ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -20,3 +20,12 @@ WM_STANDARD:
   is_canonical: true
   uses:
   - water_volume_accumulator
+
+WM_WFR:
+  description: "Standard building water meter with flow rate meter."
+  is_canonical: true
+  implements:
+  - WM_STANDARD
+  uses:
+  - water_volume_accumulator
+  - supply_water_flowrate_sensor
```

Comment:
remove `supply` so it just reads `water_flowrate_sensor`. No need for the descriptor if it is not to differentiate it from something else. Will need to be defined (I don't think it already exists).

Reply:
noted sir